### ISSUE TITLE
problem on changing the model to inception_resnet_v2

### ISSUE THREAD

**Issue by zfs1993** [Joker316701882/Additive-Margin-Softmax#27]
i change the model to inception_resnet_v2,and didnt change any other thing ,used the default parameters,it gets divergent

**Comment by zfs1993** [comment]
it is really strange,the log reveal that Loss and RegLoss is increasing during the train.but the accuracy on lfw is increasing too. so , is there anything wrong about the loss,or maybe i misunderstand the meaning of loss, what's the meaning of Loss and RegLoss?

**Comment by lostemples** [comment]
hi, did you solve your problem? @zfs1993
